Local/Devopsy.py

```diff
@@ -0,0 +1,83 @@
+import random
+import requests
+import logging
+from Legobot.Lego import Lego
+from pyquery import PyQuery
+from urllib.parse import urlencode, urljoin
+
+
+logger = logging.getLogger(__name__)
+
+
+class Devopsy(Lego):
+
+    def listening_for(self, message):
+        return message['text'].split()[0] == '!devopsy'
+
+    def handle(self, message):
+        logger.info(message)
+        try:
+            target = message['metadata']['source_channel']
+            self.opts = {'target': target}
+            self.message = message
+        except IndexError:
+            logger.error('Could not identify message source in message: {0!s}'.format(str(message)))
+        args = self.devopsy(message['text'].split())
+        self.reply(message, args, self.opts)
+
+    def devopsy(self, *args):
+        """Return a gif based on search of several *reactions
+        Ported to Legobot from:
+            "https://github.com/ricardokirkner/err-devops-reactions"
+            ... and extended.
+
+        Return a random gif if no query is specified or query not found
+
+        Example:
+            !devopsy somefoofulquery
+            !devopsy
+        """
+
+        base = {
+                'dev': 'https://devopsreactions.tumblr.com/',
+                'qa': 'https://qareacts.tumblr.com/',
+                'sec': 'https://securityreactions.tumblr.com/',
+                'dba': 'http://dbareactions.com/'
+               }
+
+        items = list(base.items())
+        key, val = random.choice(items)
```

Comment:
Add a `#nosec` comment to the end of this line per https://github.com/openstack/bandit#exclusions since it's a false positive.